package.json

```diff
@@ -53,7 +53,9 @@
     "typescript": "4.6.3"
   },
   "dependencies": {
-    "@accordproject/concerto-util": "3.9.1"
+    "@accordproject/concerto-core": "3.14.0",
+    "@accordproject/concerto-util": "3.9.1",
```

Comment:
I think adding concerto-core here creates a cyclic dependency, because concerto-core depends on the metamodel. I _think_ you are only using it in tests, so perhaps a dev dependency will work? Alternatively do what the other tests do and check in the JSON (AST) for your model and use that (created using the concerto CLI).